capa/requests.yaml

```diff
@@ -31,3 +31,7 @@ releases:
   requests:
   - name: cert-exporter
     version: ">= 2.9.2"
+- name: "> 25.1.4 < 25.2.0 || > 25.2.2 < 25.3.0 || > 25.3.1 < 25.4.0 || > 25.4.1 < 26.0.0 || > 26.1.2 < 26.2.0 || > 26.2.1 < 26.3.0 || > 26.3.1 < 27.0.0 || > 27.1.2 < 27.2.0 || > 27.2.1 < 27.3.0 || > 28.1.3 < 28.2.0 || > 28.2.1 < 28.3.0 || > 28.3.1 < 28.4.0 || > 28.4.1 < 29.0.0 || > 29.1.1 < 29.2.0 || > 29.2.1 < 29.3.0 || > 29.3.1 < 29.4.0 || > 29.4.1 < 29.5.0 || > 29.5.1 < 29.6.0"
```

Comment:
Same here: This would first affect v25.1.5, v25.2.3 and so on.

As already noted before, I'd prefer not requesting a Cilium update for these versions and just do it for v29.